include/curl/curl.h

```diff
@@ -2431,7 +2434,11 @@ CURL_EXTERN CURLcode curl_global_init_mem(long flags,
                                           curl_free_callback f,
                                           curl_realloc_callback r,
                                           curl_strdup_callback s,
-                                          curl_calloc_callback c);
+                                          curl_calloc_callback c
+#if defined(WIN32) 
+										, curl_wcsdup_callback w
+#endif
+										);
```

Comment:
This changes the API and breaks ABI! We cannot accept that. Can we wrap the function instead and do a regular malloc for it?